qkit/drivers/Keysight_VNA_E5071C.py

```diff
@@ -875,10 +877,10 @@ def do_set_active_trace(self, trace):
 
     def do_get_active_trace(self):
         """
-        :return: the active trace on the VNA
+        :return: the active trace on the VNA formatted to log amplitude
         """
-        # TODO: ask device
-        return self._active_trace
+        self._active_trace = self._visainstrument.query_binary_values('CALC{}:DATA:FDAT?'.format(self._ci), datatype='d')[0::2]
+        #return self._active_trace
```

Comment:
I think with "active_trace", the number of the currently actovated trace is meant. So you can have several lines displayed simultaneously on the screen and have to select from which one you want to get the data.
Apart from that, it looks like your function does not return anything.